mobile-app/app/screens/AppNavigator/screens/Balances/screens/AddressBookScreen.tsx

```diff
@@ -34,6 +34,9 @@ export function AddressBookScreen ({ route, navigation }: Props): JSX.Element {
   const addressBook = userPreferences.addressBook
   const [isEditing, setIsEditing] = useState(false)
 
+  // disable address selection touchableopacity from settings page
+  const disableOpacity = !!((selectedAddress === undefined && onAddressSelect === undefined))
```

Comment:
- does it work if we remove double bracket and double `!!`?
- can we rename variable to `disableAddressSelect`?

Reply:
works when removing one bracket, `!!` must stay 
yes of course we can rename it :>